grails-app/conf/application.yml

```diff
@@ -183,43 +190,65 @@ dataSource:
     pooled: true
     jmxExport: true
     driverClassName: com.mysql.cj.jdbc.Driver
-    username: 'openboxes'
-    password: 'openboxes'
+    dbCreate: none
     dialect: org.hibernate.dialect.MySQL57InnoDBDialect
-    logSql: false
-    formatSql: false
+    factory: org.apache.tomcat.jdbc.pool.DataSourceFactory
+    logger: com.mysql.cj.jdbc.log.StandardLogger
+    type: org.apache.tomcat.jdbc.pool.DataSource
     properties:
+        # from https://docs.grails.org/3.3.16/guide/conf.html#dataSource
         jmxEnabled: true
-        initialSize: 10
-        maxActive: 100
+        initialSize: 5
+        maxActive: 50
         minIdle: 5
         maxIdle: 25
         maxWait: 10000
-        maxAge: 14400
+        maxAge: 10 * 60000
         timeBetweenEvictionRunsMillis: 5000
         minEvictableIdleTimeMillis: 60000
         validationQuery: SELECT 1
         validationQueryTimeout: 3
         validationInterval: 15000
-        testOnBorrow: true
+        testOnBorrow: false
```

Comment:
testing a connection when we pull it from the pool adds a bit of latency; and since we ping unused connections every 15 seconds (testWhileIdle/validationInterval) the additional ping on checkout is unnecessary.

Reply:
I can live with that, but I'd probably rather have testOnBorrow over testWhileIdle. We can leave it as-is for now and come back to this if we encounter any issues related to the connection pool. This becomes more of a problem when the connection pool is on a separate server than the database and the cloud hosting provider is pretty cut throat when it comes to connections.